### DISCUSSION TITLE
Missing Template Bot Question

### DISCUSSION THREAD

**Discussion by koos147** [arendst/Tasmota#10501] [Q&A] [answered]
why on earth do you close any bug directly if the question isn't in a useless format?
due to a bug i lost a whole day of troubleshooting. after this it cost me a half hour to make a complete overview. 
and within a second you trow away anything like you don't care about your users.

why on earth do you put so much effort in a product when you hate your users? i simply don't understand it.

I will edit this message since answers are not allowed (again a verry rude action). 

So somebody ask a simple question. 
No information on front. 
After this you say nice question. We trow it away and you have to start over. 
Then there are 20 question 18 of them are useless and not only wasting time of the one who asks the question but also from everyone who try to answers them. 

So if you really want the questions make sure nobody can post it otherwise. The current action feels like a slap in the face.

**Comment by meingraham** [comment, marked as answer]
You expect those who are receiving the issue you're posting to take the time to read it to glean the information you've provided. Likewise, the admins are asking you to read the instructions in the template so that we have the typical information we need in order to better understand the context of your issue. So, give and take. If you cannot take a bit of time to read and provide the information requested, are we to say that you hate the admins and the other users you are asking to provide you with assistance? I don't think so.

Notice that there have been over 10K posts. This scale of interaction requires streamlining and capturing the necessary information up front in order to avoid wasting a lot of people's (_ALL_ volunteers) time.

I hope this provides some perspective from "the other side" to help you understand the reason for the bot.

Regards.

**Comment by meingraham** [comment]
🤦 

https://github.com/arendst/Tasmota/issues/10499#issuecomment-757506442
